### PR TITLE
fix(ios): show alert in background thread with native crash

### PR DESCRIPTION
**Pull request by expcapitaldev**
fix(ios): show alert in background thread

### CODE REVIEW

**Comment by expcapitaldev** [comment]
you can close my MR after adding this fix

**Comment by saggio89** [comment]
@expcapitaldev is this the bug that is throwing this error?
`Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Modifications to the layout engine must not be performed from a background thread after it has been accessed from the main thread.'`

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-sms-plugin",
-  "version": "1.0.1",
+  "version": "1.0.1-dev-1.0",
```
Comment (dbaq) [inline package.json]: I'll bump the version in another commit.
```suggestion
  "version": "1.0.1",
```

File: plugin.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
         xmlns:android="http://schemas.android.com/apk/res/android"
            id="cordova-sms-plugin"
-      version="1.0.1">
+      version="1.0.1-dev-1.0">
```
Comment (dbaq) [inline plugin.xml]: I'll bump the version in another commit.
```suggestion
      version="1.0.1">
```

**Review by dbaq — APPROVED** [review]
Thanks for the contribution. I'll merge and push a new release tonight.

**Comment by dbaq** [comment]
[Version 1.0.2](https://www.npmjs.com/package/cordova-sms-plugin/v/1.0.2) has been released. Thank you for your contribution.

**Pull request merged**